src/lib.rs

```diff
@@ -51,6 +51,25 @@ impl RadioPayloadMessage {
     }
 }
 
+// Maybe move to SDK
+lazy_static! {
+    pub static ref SUPPORTED_NETWORK_INTERVALS: HashMap<&'static str, u64> = [
+        ("goerli", 2),
+        ("mainnet", 10),
```

Comment:
Do you think this can be `eth_mainnet` to avoid confusion 🤔

Reply:
I'm following the graph-node convention to just use `mainnet`, and this connect directly to the output of graph node's `indexingStatuses` and input to `blockHashFromNumber`